examples/llava/llava-cli.cpp

```diff
@@ -243,6 +243,9 @@ int main(int argc, char ** argv) {
     }
 
     auto image_embed = load_image(ctx_llava, &params);
+    if (!image_embed) {
+        exit(1);
```

Comment:
For consistency with the rest of the returns:

```suggestion
        return 1;
```

Reply:
Done.